### PR TITLE
[IMP]10711 add module pos_printout_improvement

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

File: pos_printout_improvement/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+ 
+=====================
+Pos order improvement
+=====================
+ 
+ This module extends the functionality of pos to add customer information,
```
Comment (elicoidal) [inline pos_printout_improvement/README.rst]: s/pos/POS

File: pos_printout_improvement/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+ 
+=====================
+Pos order improvement
```
Comment (elicoidal) [inline pos_printout_improvement/README.rst]: s/Pos/POS

File: pos_printout_improvement/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+=====================
+ 
+ This module extends the functionality of pos to add customer information,
+order sequence and total quantity on the pos order printout.
```
Comment (elicoidal) [inline pos_printout_improvement/README.rst]: s/pos/POS

File: pos_printout_improvement/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Pos printout improvement',
+    'version': '8.0.1.0.1',
+    'category': 'pos',
```
Comment (elicoidal) [inline pos_printout_improvement/__openerp__.py]: s/pos/POS

**Comment by elicoidal** [comment]
small details. Other than than LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
:+1:

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**